app.js

```diff
@@ -180,7 +180,7 @@ If you encounter any issue/typo visit https://github.com/Marr11317/muscord-bot.
             });
             return;
         }
-        if (msg.content.toLowerCase().includes("interesting") && msg.author.id !== client.user.id) {
+        if (msg.content.toLowerCase().includes("interesting...") && msg.author.id !== client.user.id) {
```

Comment:
Haha that was so funny actually. Do you think peole say interesting... that often? Okay maybe it's only funny the first time...

Reply:
After thinking a bit more, you are right that it will happen too often.